app/models/arbitrary_assignment_creation.rb

```diff
@@ -20,6 +23,10 @@ def self.create!(params)
     new(params).tap(&:save!)
   end
 
+  def force
+    instance_variable_defined?(:@force) ? @force : false
```

Comment:
if you removed the condition above then this could be `@force == true`, couldn't it?

Reply:
I'm interested in validating that this is not passed as null